### PR TITLE
BACKLOG-21975 : Fix jahia 8.1.3.0 compatibilty

### PR DESCRIPTION
**Pull request by dgriffon**
Make sitemap compatible with Jahia 8.1.3.0
This required to set fixed version for:

```
    "@jahia/data-helper": "1.1.2",
    "@jahia/ui-extender": "1.1.0",
    "@jahia/react-material": "3.0.5",
```

### CODE REVIEW

File: src/main/resources/META-INF/definitions.cnd [hunk 1]
```diff
@@ -8,6 +8,7 @@
  - sitemapIndexURL (string) = ''  indexed=no hidden autocreated
  - sitemapHostname (string) = ''  indexed=no hidden autocreated
  - sitemapCacheDuration (string) = ''  indexed=no hidden autocreated
+ - isJobTriggered (boolean)  indexed=no hidden
```
Comment (jsinovassin) [inline src/main/resources/META-INF/definitions.cnd:11]: Do we use this property somewhere?

File: src/javascript/components/SitemapPanelApp.jsx [hunk 2]
```diff
@@ -155,6 +155,7 @@ const SitemapPanelApp = ({client, t}) => {
     }
 
     return (
+
```
Comment (jsinovassin) [inline src/javascript/components/SitemapPanelApp.jsx:158]: Seems to be an unnecessary  space

File: src/main/resources/META-INF/definitions.cnd (hunk 1 above)
Comment (dgriffon) [reply src/main/resources/META-INF/definitions.cnd]: no, it was for testing purpose, thank you!

File: src/javascript/components/SitemapPanelApp.jsx (hunk 2 above)
Comment (dgriffon) [reply src/javascript/components/SitemapPanelApp.jsx:157]: removed

**Review by jsinovassin — APPROVED** [review]

**Pull request merged**